python-regression/tests/features/steps/api_test_steps.py

```diff
@@ -228,3 +232,63 @@ def check_node_sync(step, node, milestone):
         "Latest Solid Milestone {} on {} is not the expected {}".format(latestSolidMilestone, node, milestone)
 
 
+@step(r'A transaction is issued on "([^"]+)" with:')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['step' may be undefined, or defined from star imports: aloe](https://app.codacy.com/app/GalRogozinski/iri/pullRequest?prid=4517274)